File: ORB.Core/Models/CorporateRiskUpdate.cs
using System;
using System.Collections.Generic;

namespace ORB.Core.Models
{
    public partial class CorporateRiskUpdate : RiskUpdate
    {
        public CorporateRiskUpdate() : base()
        {
            RiskMitigationActionUpdatesNavigation = new HashSet<CorporateRiskMitigationActionUpdate>();
        }

        public virtual CorporateRisk Risk { get; set; }
        public virtual ICollection<CorporateRiskMitigationActionUpdate> RiskMitigationActionUpdatesNavigation { get; set; }
    }
}

File: corporate-reporting/src/types/CorporateRiskUpdate.ts
import { ICorporateRisk } from "./CorporateRisk";
import { IRiskUpdate, RiskUpdate } from "./RiskUpdate";

export interface ICorporateRiskUpdate extends IRiskUpdate {
    Risk?: ICorporateRisk;
}

export class CorporateRiskUpdate extends RiskUpdate implements ICorporateRiskUpdate { }